src/System.Private.ServiceModel/tools/IISHostedWcfService/Web.config

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0" encoding="utf-8"?>
+ <!--Copyright (c) Microsoft Corporation.  All Rights Reserved.-->
+<configuration>
+  <system.diagnostics>
+    <sources>
+      <source name="System.ServiceModel" switchValue="Information, ActivityTracing" propagateActivity="true">
+        <listeners>
+          <add name="traceListener" type="System.Diagnostics.XmlWriterTraceListener" initializeData="c:\log\Traces.svclog" />
```

Comment:
Good conversation but do we need to get svclog tracing working on this PR?   It feels like a lower-priority issue to deal with later when everything is running.

---
In reply to: [59334362](https://github.com/dotnet/wcf/pull/995#discussion_r59334362) [](ancestors = 59334362)